plugins/connection/chroot.py

```diff
@@ -7,7 +7,7 @@
 # GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-from __future__ import (absolute_import, division, print_function)
+from __future__ import annotations
 __metaclass__ = type
```

Comment:
Do you want to do this in the same PR? This is also no longer needed.
```suggestion
```

Reply:
done